fluentd-gcp-scaler/scaler.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+SLEEP_SECONDS=${SLEEP_SECONDS:-60}
+NAMESPACE=${NAMESPACE:-kube-system}
+DEFAULT_CPU_REQUEST=${CPU_REQUEST}
+DEFAULT_MEMORY_REQUEST=${MEMORY_REQUEST}
+DEFAULT_CPU_LIMIT=${CPU_LIMIT}
+DEFAULT_MEMORY_LIMIT=${MEMORY_LIMIT}
+
+reset_to_defaults() {
+  REQUESTS=
+  LIMITS=
+  CPU_REQUEST=${DEFAULT_CPU_REQUEST}
+  MEMORY_REQUEST=${DEFAULT_MEMORY_REQUEST}
+  CPU_LIMIT=${DEFAULT_CPU_LIMIT}
+  MEMORY_LIMIT=${DEFAULT_MEMORY_LIMIT}
+}
+
+apply_scaling() {
+  if ! kubectl get scalingpolicies -n ${NAMESPACE} ${SCALING_POLICY}
```

Comment:
Since ScalingPolicy is not a regular k8s resource, please add a comment and link to the CRD definition.

Reply:
Added.